data/json/items/gun/50.json

```diff
@@ -67,7 +67,9 @@
     },
     "material": [ "steel", "wood" ],
     "clip_size": 1,
+    "modes": [],
```

Comment:
Drop as in delete it? Because that crashes my game.
Would `manual` be okay, since the description mentions altering feeding mechanism and gives off this whole DIY feel?

```
"clip_size": 1,
"modes": [ [ "DEFAULT", "manual", 1 ]
```